ext/drx/drx.c

```diff
@@ -2520,6 +2520,22 @@ drx_expand_scatter_gather(void *drcontext, instrlist_t *bb, OUT bool *expanded)
  * as well as AVX2 gather.
  */
 
+typedef struct _drx_state_machine_params_t {
+    byte *pc;
+    byte *prev_pc;
+    int detect_state;
+    byte *restore_dest_mask_start;
+    byte *restore_scratch_mask_start;
```

Comment:
What are these?  Are they PC values?  How about putting that in the name: s/start/start_pc/ for both.

Reply:
Good idea, done.